infrastructure-playbooks/lv-create.yml

```diff
@@ -0,0 +1,58 @@
+- name: Creates Logical volumes for the Bucket Index or FS Journals on a single device. Prepares for use of osd_scenario=lvm. 
+  hosts:
+  - osds
+
+  tasks:
+
+  - name: Include vars of lv_vars.yaml
+    include_vars:
+      file: lv_vars.yaml
+
+  # need to check if lvm2 is installed
+  - name: install lvm2
+    package:
+      name: lvm2
+      state: present
+
+  # Make entire device a VG
+  - name: add device as LVM PV
+    lvg:
+      force: yes
+      pvs: "{{device}}"
+      pesize: 4
+      state: present
+      vg: "{{vg_name}}"
+
+  # Create LVs 
+  - name: create LVs for journals
+    lvol:
+      lv: "{{item.journal_name}}"
+      vg: "{{vg_name}}"
+      size: "{{item.journal_size}}"
+      pvs: "{{device}}"
+    with_items:
+      - "{{ lvs }}"
+
+  - name: create LVs for data (ex: bucket index)
+    lvol:
+      lv: "{{item.data_lv_name}}"
+      vg: "{{vg_name}}"
+      size: "{{item.data_lv_size}}"
+      pvs: "{{device}}"
+    with_items:
+      - "{{ lvs }}"
+
+  # Write ending configuration logfile
+  - name: save LV configuration
+    shell: lvs -o -origin,convert_lv,pool_lv,move_pv,mirror_log,data_lv,lv_metadata_size,data_percent,metadata_percent,copy_percent,lv_attr
+    run_once: true
+    register: lvs_out
+
+  - name: write output for osds.yml to logfile
+    action: template src=templates/logfile.j2 dest=/tmp/logfile.txt
```

Comment:
This will overwrite the log file for each osd host. I'm thinking we'd want osd specific logs.

Also, can we not use ``action`` here and format ``template`` like the rest of the tasks?